gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/resources/GenerateResClassTask.kt

```diff
@@ -9,13 +9,12 @@ import org.jetbrains.compose.internal.IdeaImportTask
 import java.io.File
 
 internal abstract class GenerateResClassTask : IdeaImportTask() {
-    companion object {
-        private const val RES_FILE_NAME = "Res"
-    }
-
     @get:Input
     abstract val packageName: Property<String>
 
+    @get:Input
+    abstract val resClassName: Property<String>
```

Comment:
Should we name it just `className`? To be consistent with `packageName` and to avoid double wording in `resources { res... = ``

Reply:
I don't think so, because it changes the `Res` class name only.

Reply:
for the consistency I'll rename it to `nameOfResClass`